test/expected/views.out

```diff
@@ -0,0 +1,99 @@
+SELECT * FROM information_schema._timescaledb_hypertables;
+ table_schema | table_name |    table_owner    | num_dimensions | num_chunks | table_size | index_size | toast_size | total_size 
+--------------+------------+-------------------+----------------+------------+------------+------------+------------+------------
+ public       | ht1        | default_perm_user |              1 |          1 | 8192 bytes | 16 kB      |            | 24 kB
+ public       | ht2        | default_perm_user |              1 |          1 | 8192 bytes | 16 kB      | 8192 bytes | 32 kB
+(2 rows)
+
+\c single :ROLE_SUPERUSER
+-- create schema open and hypertable with 3 chunks
+CREATE SCHEMA open;
+GRANT USAGE ON SCHEMA open TO :ROLE_DEFAULT_PERM_USER;
+CREATE TABLE open.open_ht(time TIMESTAMPTZ NOT NULL);
+SELECT create_hypertable('open.open_ht','time');
+ create_hypertable  
+--------------------
+ (3,open,open_ht,t)
+(1 row)
+
+INSERT INTO open.open_ht SELECT '2000-01-01'::TIMESTAMPTZ;
+INSERT INTO open.open_ht SELECT '2001-01-01'::TIMESTAMPTZ;
+INSERT INTO open.open_ht SELECT '2002-01-01'::TIMESTAMPTZ;
+-- create schema closed and hypertable
+CREATE SCHEMA closed;
+CREATE TABLE closed.closed_ht(time TIMESTAMPTZ NOT NULL);
+SELECT create_hypertable('closed.closed_ht','time');
+   create_hypertable    
+------------------------
+ (4,closed,closed_ht,t)
+(1 row)
+
+INSERT INTO closed.closed_ht SELECT '2000-01-01'::TIMESTAMPTZ;
+SELECT * FROM information_schema._timescaledb_hypertables;
+ table_schema | table_name |    table_owner    | num_dimensions | num_chunks | table_size | index_size | toast_size | total_size 
+--------------+------------+-------------------+----------------+------------+------------+------------+------------+------------
+ public       | ht1        | default_perm_user |              1 |          1 | 8192 bytes | 16 kB      |            | 24 kB
+ public       | ht2        | default_perm_user |              1 |          1 | 8192 bytes | 16 kB      | 8192 bytes | 32 kB
+ open         | open_ht    | super_user        |              1 |          3 | 24 kB      | 48 kB      |            | 72 kB
+ closed       | closed_ht  | super_user        |              1 |          1 | 8192 bytes | 16 kB      |            | 24 kB
+(4 rows)
+
+\c single :ROLE_DEFAULT_PERM_USER
+SELECT * FROM information_schema._timescaledb_hypertables;
+ table_schema | table_name |    table_owner    | num_dimensions | num_chunks | table_size | index_size | toast_size | total_size 
+--------------+------------+-------------------+----------------+------------+------------+------------+------------+------------
+ public       | ht1        | default_perm_user |              1 |          1 | 8192 bytes | 16 kB      |            | 24 kB
+ public       | ht2        | default_perm_user |              1 |          1 | 8192 bytes | 16 kB      | 8192 bytes | 32 kB
+ open         | open_ht    | super_user        |              1 |          3 | 24 kB      | 48 kB      |            | 72 kB
```

Comment:
In addition to which schema, I think the other important issue to resolve in this PR is permissions. It looks like the current iteration still has incorrect permissions, since`default_perm_user` can still view hypertables that belong to the `super_user`.

Reply:
What do you mean by incorrect permissions? This query runs with the permissions of the user so any additional filtering we do can be easily circumvented. The user won't be able to view storage information for hypertables he doesnt have permissions for but the hypertables will still be listed, this was what @cevian and I agreed on. Do you want me to filter those out, even though the user could just query the catalog directly where there is no filtering.

Reply:
Nevermind, agree offline that this is allowed.